server/container_stats_list.go

```diff
@@ -31,13 +33,20 @@ func (s *Server) ListContainerStats(ctx context.Context, req *types.ListContaine
 	for _, container := range ctrList {
 		sb := s.GetSandbox(container.Sandbox())
 		if sb == nil {
-			log.Warnf(ctx, "unable to get stats for container %s: sandbox %s not found", container.ID(), container.Sandbox())
+			// Because we don't lock, we will get situations where the container was listed, and then
+			// its sandbox was deleted before we got to checking its stats.
+			// We should not log in this expected situation.
```

Comment:
IDK, any thought on keeping the old log message but converting to a debugf?

Reply:
I thought about it, but I don't really think it's useful. the sandbox not existing anymore isn't an actionable error as the others *may* be (though likely aren't). If sandboxes are disappearing underneath us, many other things will break much louder